### PR TITLE
Development

### PR DESCRIPTION
**Pull request by dineshkumar-2003**
- Added assignment 1 with list , tuples and dictionaries and its built in methods .
- Added assignment 2 with nested dictionary ,sets and its built in methods .
- Added assignment 3 with class , objects , exception , if else and match case .
- Added assignment 4 with two classes along with the methods .
-

### CODE REVIEW

File: python_as1.txt
Comment (praveen-pb) [inline python_as1.txt:1]: Don't add text file. add it with .py extension

File: python_as2.txt [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+1
+2
+3
+4
+5
+6
+
+#for loop with continue --it is used to skip a particular iteration
+
+for itr in range(11):
```
Comment (praveen-pb) [inline python_as2.txt]: to understand the continue concept, write more code and use continue in the middle of the code. continue stops executing the further code only for that specific iteration.

File: python_as3.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+class Calculator:
+
+    def __init__(self,num_1,num_2): #using __init__ to initialize the variables
+        self.num_1=num_1
+        self.num_2=num_2
+
+
```
Comment (praveen-pb) [inline python_as3.py]: inside a class, only one line space is enough. outside the class you need to have 2 lines of space.

File: python_as3.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    def division(self,num_1,num_2):
+        try:        #Using try and except
+            ans=num_1//num_2
+        except ZeroDivisionError as e:
+            print("Divide by zero is not possible :",e)
+
+
+
+obj_1=Calculator(10,20)  #Creating the object for the class
```
Comment (praveen-pb) [inline python_as3.py:34]: Please do not use Calculator again. come up with good example and write more logics inside every function. Add tuples, list, dict, set inside the function.

File: python_as4.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
+class BankAccount:                  #Parent class for bank account
```
Comment (praveen-pb) [inline python_as4.py:1]: Good to see a class which is not Calculator. But add more logics to the code. add try exception or if condition check

File: python_as4.py [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+print("Welcome to the ATM ")
+
+# Generating the 4 digits random number
+otp=random.randint(1000,9999)
+print('otp for bank operations is :',otp)
+
+#Object creation
+customer=BankActions(otp)
+while(flag==1):
```
Comment (praveen-pb) [inline python_as4.py:70]: Put the business logic inside any of the function

File: python_as1.txt
Comment (dineshkumar-2003) [reply python_as1.txt:1]: yes sir

File: python_as3.py (hunk 2 above)
Comment (dineshkumar-2003) [reply python_as3.py]: Ok sir. I had made the changes

File: python_as2.txt (hunk 1 above)
Comment (dineshkumar-2003) [reply python_as2.txt]: ok sir

File: python_as3.py (hunk 3 above)
Comment (dineshkumar-2003) [reply python_as3.py:34]: Ok sir I have added College class in the same file

File: python_as4.py (hunk 4 above)
Comment (dineshkumar-2003) [reply python_as4.py:1]: Ok sir

File: python_as4.py (hunk 5 above)
Comment (dineshkumar-2003) [reply python_as4.py:70]: ok sir I have added the business logics

**Review by praveen-pb — APPROVED** [review]

**Pull request merged**